exit-status.md

```diff
@@ -0,0 +1,67 @@
+---
+RFC: unassigned
+Author: Nicholas Carpenter <ncarpenter@ebsco.com>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Signal outside tools of specific Chef-Client run status.  Ability to determine results of a Chef-Client run.
+
+## Motivation
+    As a Chef user,
+    I want to be able to determine when a chef-client run is rebooting the node,
+    so that Test-Kitchen/Vagrant/any outside tool can wait for node to reboot, and continue converging.
+    
+## Specification
+* Chef-apply, Chef-client, Chef-Solo should honor the below exit chef run exit codes
+
+### Exit codes reserved across platforms
```

Comment:
Reserved by who? Chef? The links make it obvious if you read them, but it'd be nice if the header indicated to most people not to bother reading those links.

Exit Codes Reserved by the System?